frontend/scripts/router/nav-links.js

```diff
@@ -55,6 +55,11 @@ let nav = [
   {
     name: 'About',
     page: 'about'
+  },
+  {
+    name: 'Trase Finance',
```

Comment:
```suggestion
    name: 'Finance',
```

This should be just "Finance". Not sure if that affects the widths elsewhere?